sys-kernel/asahi-sources/asahi-sources-6.13.7_p4.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 2023-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+ETYPE="sources"
+
+CKV="$(ver_cut 1-3)"
+K_SECURITY_UNSUPPORTED="1"
+K_WANT_GENPATCHES="base extras"
+K_GENPATCHES_VER="9"
+K_NODRYRUN="1"
+
+RUST_MIN_VER="1.84.0"
+RUST_USEDEP='rust-src,rustfmt'
```

Comment:
RUST_USEDEP doesn't seem to be a thing anymore, even though the old version still uses it apparently. Overlay uses the current `RUST_REQ_USE` for this, but also a lower rust minimum?

Reply:
See 9224c0acf8aa541c49de424042c88e87f0e76e26.